aw_server/custom_watcher_pages.py

```diff
@@ -0,0 +1,44 @@
+import logging
+
+from flask import Blueprint, send_from_directory, jsonify, request, current_app, escape
+from iso8601 import iso8601
+
+
+def get_bucket_name_from_watcher_name(buckets, watcher_name: str):
+    for bucket in buckets:
+        if bucket.startswith(watcher_name):
+            return bucket
+
+    logging.warning(f"Cannot find bucket for watcher {watcher_name}")
+    return None
+
+
+def get_custom_watcher_blueprint(custom_watcher_static_directories):
```

Comment:
Maybe call this simply the `get_custom_static_blueprint`? (and make similar naming changes across the file to simplify and generalize)

Reply:
Okay.